README.md

```diff
@@ -58,3 +58,8 @@ fn main() -> Result<(), Error> {
     Ok(())
 }
 ```
+### MacOS
+
+For the time being, due to quirks of the total toolchain, MacOS is unsupported. `neptune-triton` will build on MacOS and
```

Comment:
Should probably not reference `neptune-triton` here

Reply:
Whoops, you're right. Fixed.